HW20/group_vars/ubuntu/ubuntu.yaml

```diff
@@ -0,0 +1,3 @@
+ansible_user: vagrant
```

Comment:
Put those variable to another "general" group OR to all hosts group

Reply:
Sounds good, doesn't work.